hash_reader.go

```diff
@@ -16,10 +16,9 @@
 package ionhash
 
 import (
+	"ion-go/ion"
```

Comment:
This is a third party library so it needs to be on its own line, please revert this

Reply:
Done.